src/gei/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -131,7 +131,7 @@ public async Task Handle(MigrateRepoCommandArgs args)
             var shouldUploadOnce = args.GitArchivePath == args.MetadataArchivePath;
 
             args.GitArchiveUrl = await UploadArchive(
-                args.GithubSourceOrg,
+                args.GithubTargetOrg,
```

Comment:
We should probably have a test for this :smile:

Reply:
Adding an integration test for using github owned storage w/ local file would be 👍

Reply:
We don't necessarily need an INT test here this as it can be done via a simple unit test. All that matters is to make sure we're using the target org [here](https://github.com/github/gh-gei/blob/main/src/gei/Commands/MigrateRepo/MigrateRepoCommandHandler.cs#L374)